packages/dd-trace/src/appsec/waf/waf_manager.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+const log = require('../../log')
+const Reporter = require('../reporter')
+const addresses = require('../addresses')
+const web = require('../../plugins/util/web')
+const { storage } = require('../../../../datadog-core')
+const WAFContextWrapper = require('./waf_context_wrapper')
+
+const validAddressSet = new Set(Object.values(addresses))
+const WAF_ENGINE_CONTEXT_KEY = Symbol('WAF_ENGINE_CONTEXT')
+// TODO MAX_CONTEXT_SIZE = 1024
```

Comment:
What's the meaning of this TODO?

Reply:
We shouldn't send to waf more than 1024 addresses, but it is not possible because we only have 13 addresses. It was checked in old approach, but I haven't added it here.

Reply:
I'm removing it